src/api/profile.ts

```diff
@@ -48,3 +48,17 @@ export const makeUpdateUserGridLocation = ({
       firestore.doc(doc).set(newData);
     });
 };
+
+export interface makeUpdateUserOnlineStatusProps {
+  userUid: string;
+}
+
+export const makeUpdateUserOnlineStatus = ({
+  userUid,
+}: makeUpdateUserOnlineStatusProps) => (status: string) => {
```

Comment:
If it’s a ‘make’ function it’s curried so that the first parameter can be bound when it’s passed to something like useCallback/similar. Is that what this is being used for? If not, the ‘make’ prefix and currying may not make sense